lib/api/compiler/class-beans-compiler.php

```diff
@@ -476,6 +476,10 @@ public function get_remote_content() {
 			$request  = wp_remote_get( $fragment );
 		}
 
+		if ( is_wp_error( $request ) ) {
```

Comment:
The error can only occur when we go into the if block above. Right?  Why? Because we are checking for the error before the if block.  Therefore, to avoid a double check, move this one into the if block above.

Reply:
Right. Good call. Done.